appveyor.yml

```diff
@@ -37,11 +40,9 @@ build_script:
   - bash build.sh
   - cd docs
   - bash builddocs.sh
-
-# TODO: Reinstate these...
-#  - bash runcoverage.sh
-#  - bash coveralls.sh
-
+  - cd ..
+  - bash runcoverage.sh
```

Comment:
good question.  this is where docs are created and where code coverage was run previously.